src/isyntax_example.c

```diff
@@ -1,38 +1,90 @@
-#include "common.h"
+#include "libisyntax.h"
+
+#include <stdint.h>
+#include <assert.h>
+
+#define STB_IMAGE_WRITE_IMPLEMENTATION
+#include "third_party/stb_image_write.h"  // for png export
+
 
-#include "isyntax.h"
 
 #define LOG_VAR(fmt, var) printf("%s: %s=" fmt "\n", __FUNCTION__, #var, var)
 
+uint32_t bgra_to_rgba(uint32_t val) {
```

Comment:
Maybe we can add a specifier for the desired pixel format in the API, so that conversion won't be required? See for example this: 
https://github.com/amspath/slidescape/blob/6f89875e69c1246b7ae5c88d3819c2f3a1ca6349/src/core/image.c#L731

To have iSyntax output to RGBA instead of BGRA, we can add a RGBA variant for `convert_ycocg_to_bgra_block()` in isyntax.c:
https://github.com/amspath/libisyntax/blob/main/src/isyntax/isyntax.c#L1334